docs/_posts/2016-05-23-standards-are-the-future.md

```diff
@@ -0,0 +1,55 @@
+
+## ES6 Classes
+ES6 Classes, also known as “ES2015 Classes” or “Javascript Classes”, are the newly-introduced javascript standard for representing classes.  React components can be defined using standard classes, instead of using `React.createClass`.
+
+A React component defined using the class syntax looks like this:
+
+```js
+class MyComponent extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return <div>Hello {this.props.name}</div>;
+  }
+  componentDidMount() {
+    // we can override a lifecycle method.
+  }
+  componentDidUpdate() {
+    // we can override another lifecycle method.
+  }
+}
+```
+
+As you can see, a component defined using a native javascript class is fairly similar to a component using the old `createClass` syntax.  You can still override the same lifecycle methods, you can still access `this.props` and `this.state`, etc.
+
+However, there are a few differences.  For instance, the `isMounted` method no longer exists ([isMounted was an antipattern anyway](/react/blog/2015/12/16/ismounted-antipattern.html)), mixins are no longer supported (having been [replaced by composition](https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750#.8552ezwh9)), etc.
+
+You can get an overview of the differences in our [React 0.13-beta blog post](/react/blog/2015/01/27/react-v0.13.0-beta-1.html) and our [Reusable Components documentation](/react/docs/reusable-components.html#es6-classes).  If you are migrating your existing codebase from `createClass` to standard javascript classes, you might find [Kris Jordan’s Migration Guide](http://www.newmediacampaigns.com/blog/refactoring-react-components-to-es6-classes) helpful.
+
+## Stateless Functional Components
+If your components are stateless and functional (ideally, most of your components should be), then you can define your component as a simple function.  A Stateless Functional Component (sometimes referred to as an “SFC” on the internet), is just a render function that takes in props and returns JSX.
+
+An example component looks like this:
+
+```js
+function MyComponent(props) {
+  return <div>Hello {props.name}!</div>
+}
+```
+
+Stateless functional components are a convenient syntax for lightweight components.  They can also be optimized within the React core to follow slightly faster code paths and use less memory.  However, they don’t have a backing instance, so you can’t attach a ref to the component, and there is no `this` object upon which to store internal state.
```

Comment:
“they can also be optimized”—let’s avoid setting the wrong expectation, people have been bitten by something like this where we say they will eventually get faster in theory, and people take it as if something like `PureRenderMixin` gets applied to them automatically, which is far from truth.